src/coreclr/interpreter/compiler.cpp

```diff
@@ -461,7 +461,10 @@ void InterpCompiler::LinkBBs(InterpBasicBlock *from, InterpBasicBlock *to)
         if (newCapacity > prevCapacity)
         {
             InterpBasicBlock **newa = (InterpBasicBlock**)AllocMemPool(newCapacity * sizeof(InterpBasicBlock*));
-            memcpy(newa, from->ppOutBBs, from->outCount * sizeof(InterpBasicBlock*));
+            if (prevCapacity != 0) 
```

Comment:
It would be more readable to use the `from->outCount` instead of the prevCapacity. The way you have it, the reader of this code would need to know how the `prevCapacity` is computed.